package-lock.json

```diff
@@ -0,0 +1,7268 @@
+{
+  "name": "jsonwebtoken",
```

Comment:
It seems the package-lock wasn't committed before which is not the best practice. 
It might be better to add it in a different commit for visibility